src/main/java/dan200/computercraft/core/filesystem/FileSystem.java

```diff
@@ -730,7 +730,7 @@ private MountWrapper getMount( String path ) throws FileSystemException
         }
         if( match == null )
         {
-            throw new FileSystemException( "Invalid Path" );
+            throw new FileSystemException( "Invalid Path (" + path + ")" );
```

Comment:
Shouldn't this be `"/" + path + ": Invalid path"` to be consistent with the others?